microsoft-edge/dev-videos/index.md

```diff
@@ -29,6 +29,22 @@ To add a new video:
 -->
 
 
+<!-- ====================================================================== -->
+## Web Platform - Create a scroll-linked animation without JavaScript
+
+_Published on 12 May 2022._
+
+[![Thumbnail image for the scroll-linked animation API video](./images/scroll-linked-animations.png)](https://www.youtube.com/watch?v=Q0nhiHVVnvI)
+
+Learn about the upcoming CSS scroll-linked animations feature and how it can be used to create a reading progress indicator on a web page without JavaScript.
```

Comment:
```suggestion
Learn about the upcoming _scroll-linked animations_ feature of CSS and how it can be used to create a reading progress indicator on a webpage, without using JavaScript.
```